python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/errdisable.schema.yml

```
@@ -16,12 +16,28 @@ keys:
             type: list
             items:
               type: str
+              description: |-
+                Specifies the events that should trigger this action.
+                The list of supported causes is built dynamically from Sysdb and
+                depends on both the EOS version and the hardware platform. Your device
+                may not support all causes shown here.
```

Let's keep it simple (I know I wrote more in our chat :) )

```suggestion
                The list of supported causes depends on both the EOS version and the hardware platform.
```